tests/integration_tests/functional/test_api.py

```diff
@@ -181,7 +181,7 @@ def test_api_put_update_pre_boot(test_microvm_with_api):
     # The machine configuration has a default value, so all PUTs are updates.
     microvm_config_json = {
         'vcpu_count': 4,
-        'ht_enabled': True,
+        'smt': not platform.machine() == 'aarch64',
```

Comment:
Why not `platform.machine() == 'x86_64'`?

Reply:
done